src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {
+    Component,
+    ElementRef,
+    EventEmitter,
+    Input,
+    Output,
+    ViewChild
+} from "@angular/core";
+import {TabContent} from "../tabs/tab-content";
+
+@Component({
+    selector: "clr-wizard-page",
+    templateUrl: "./wizard-page.html",
+    host: {
+        "[id]": "id",
+        "[class.clr-nav-content]": "true",
+        "role": "tabpanel",
+        "[attr.aria-hidden]": "!active",
+        "[attr.aria-labelledby]": "ariaLabelledBy",
+        "[attr.data-hidden]": "!active",
+        "[class.active]": "active"
+    }
+})
+
+export class WizardPage extends TabContent {
+    @ViewChild("titleContainer") titleContainer: ElementRef;
+
+    // title of the page, assigned by Wizard if this component doesn't have projected title content
+    title: string;
+
+    hasProjectedTitleContent: boolean = false;
+
+    // User can bind his event handler for onCommit of the main content
+    @Output("clrWizardPageOnCommit") onCommit: EventEmitter<any> =
+        new EventEmitter<any>(false);
+
+    // User can bind his/her event handler for onLoad of the main content
+    @Output("clrWizardPageOnLoad") onLoad: EventEmitter<any> = new EventEmitter(false);
+
+    // input variable, optional, to set if this tab is skipped
+    @Input("clrWizardPageIsSkipped") isSkipped: boolean = false;
+
+    // input variable, optional, to set if this tab is skipped
+    @Input("clrWizardPagePreventDefault") preventDefault: boolean = false;
+
+    // Next button disabled
+    @Input("clrWizardPageNextDisabled") public nextDisabled: boolean;
+
+    // Error Flag Raised
+    @Input("clrWizardPageErrorFlag") public errorFlag: boolean;
+
+    // Emitter for Next button state changes
+    @Output("clrWizardPageNextDisabledChanged") nextDisabledChanged: EventEmitter<any> =
+        new EventEmitter(false);
+
+    constructor() {
+        super();
+    }
+
+    ngOnInit() {
+        let projectedTitleHTML: string = this.titleContainer.nativeElement.innerHTML.trim();
```

Comment:
Hmm I wasn't aware of limitations wrt accessing a native element's property in AoT. That being said, I tested the components in AoT, and they are working as expected. Let me know if you have any specific documentation or issue that you saw this in and I can take a look.